src/v/pandaproxy/schema_registry/protobuf.cc

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2021 Vectorized, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#include "pandaproxy/schema_registry/protobuf.h"
+
+#include <fmt/ostream.h>
+#include <google/protobuf/descriptor.h>
+
+namespace pandaproxy::schema_registry {
+
+namespace {
+
+namespace pb = google::protobuf;
+
+}
```

Comment:
i'm not familiar with this pattern. wouldn't `pb` already have file-scope without the anonymous namespace?

Reply:
> i'm not familiar with this pattern. wouldn't `pb` already have file-scope without the anonymous namespace?

It's not necessary. A later commit adds stuff to this namespace, it's an artifact of me carving up the commits.

Reply:
Cool